main/py-virtualenv/APKBUILD

```diff
@@ -8,21 +9,50 @@ pkgdesc="Virtual Python Environment builder"
 url="http://www.virtualenv.org/en/latest/"
 arch="noarch"
 license="MIT"
-depends="python2 py2-pip"
-makedepends="python2-dev"
+depends=""
+makedepends="python2-dev python3-dev" # add back py-tox for check()
+subpackages="py2-${pkgname#py-}:_py2 py3-${pkgname#py-}:_py3"
 source="https://files.pythonhosted.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz"
 builddir="$srcdir/$_pkgname-$pkgver"
 
+options="!check" # circular dep, this pkg is need for as py3 before "!check" can be removed
+
 build() {
 	cd "$builddir"
-	python2 setup.py build || return 1
+
+	python2 setup.py build
+	python3 setup.py build
+}
+
+check() {
+  cd "$builddir"
```

Comment:
Please use only tabs for indentation (and alignment) in APKBUILDs and other shell scripts, not spaces. If you install http://editorconfig.org/ to your editor, it’ll take care of it for you.